MobileSyncExplorerKotlinTemplate/template.js

```diff
@@ -31,46 +31,65 @@
  *   workspace
  *   bootconfigFile
  */
+
+const path = require('path');
+const fs = require('fs');
+
+function listKtFiles(dirPath) {
+    var result = [];
+    fs.readdirSync(dirPath).forEach(function (file) {
+        const fileFullPath = path.join(dirPath, file);
+        const stats = fs.statSync(fileFullPath);
+        if (fs.statSync(fileFullPath).isDirectory()) {
+            result = result.concat(listKtFiles(fileFullPath));
+        } else {
+            if (file.endsWith('.kt')) {
+                result.push(fileFullPath);
+            }
+        }
+    });
+    return result;
+}
+
+
 function prepare(config, replaceInFiles, moveFile, removeFile) {
 
-    var path = require('path');
+    // Values in template
+    const templateAppName = 'Mobile Sync Explorer Kotlin Template';
+    const templatePackageName = 'com.salesforce.mobilesyncexplorerkotlintemplate';
+    const templatePackagePath = templatePackageName.replaceAll(".", path.sep);
+    const configPackagePath = config.packagename.replaceAll(".", path.sep);
+    
+    // Key files
+    const templatePackageJsonFile = 'package.json';
+    const templateSettingsGradle = 'settings.gradle';
+    const templateAndroidManifestFile = path.join('app', 'AndroidManifest.xml');
+    const templateStringsXmlFile = path.join('app', 'src', 'main', 'res', 'values', 'strings.xml');
+    const templateBootconfigFile = path.join('app', 'src', 'main', 'res', 'values', 'bootconfig.xml');
+    const javaDirPath = path.join('app', 'src', 'main', 'java');
+    const ktFiles = listKtFiles(javaDirPath);
 
     //
-    // FIXME - move source files according to user provided app package
+    // TODO rename application class
```

Comment:
Is this still relevant, or should it be removed?

Reply:
It should be removed. 
Done in https://github.com/forcedotcom/SalesforceMobileSDK-Templates/pull/341/commits/59f47d60c05a5e1760584ab2ef3607a1c2719a1b